stories/storybook/styles.scss

```diff
@@ -261,38 +263,53 @@ code {
   .tag {
     color: mercury.$color-teal;
   }
-}
-
-// CSS
-[lang='scss'] {
-  .rule {
+  .function {
+    color: mercury.$color-orange;
+  }
+  .keyword {
     color: mercury.$color-purple;
   }
-  .selector {
-    color: mercury.$color-teal;
+  .parameter {
+    color: #ea62b3;
   }
   .string {
     color: mercury.$color-orange;
   }
 }
 
-// JS
-[lang='js'] {
-  .function {
-    color: mercury.$color-orange;
+// CSS
+.language-scss {
+  .comment {
+    color: mercury.$color-gray;
   }
   .keyword {
     color: mercury.$color-purple;
   }
-  .parameter {
-    color: #ea62b3;
+  .rule {
+    color: mercury.$color-purple;
+  }
+  .selector {
+    color: mercury.$color-teal;
   }
   .string {
     color: mercury.$color-orange;
   }
 }
 
-// GitHub tables
+// GitHub
+
+kbd {
```

Comment:
Good call adding this, I'll use it for options tables in other components too. 🎉

Reply:
We put this on the docs site (and www): https://github.com/sindresorhus/github-markdown-css. I make a habit out of copying stuff out of there from time to time and–hey we should do that for other stuff